### PR TITLE
Add integrity attribute to script used by CNCF Landscape shortcode 

### PR DESCRIPTION
**Pull request by dev-johnn**
Add integrity attribute to script used by CNCF Landscape shortcode.

Fixes [#45404](https://github.com/kubernetes/website/issues/45404#issue-2163352174)

### CODE REVIEW

File: layouts/shortcodes/cncf-landscape.html [hunk 1]
```diff
@@ -57,7 +57,7 @@
 </script>
 {{- end -}}
 <div id="frameHolder">
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/iframe-resizer/4.3.9/iframeResizer.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/iframe-resizer/4.3.9/iframeResizer.min.js" intregrity="sha384-vtXRMe3mGCbOeY7l30aIg8H9p3GdeSe4IFlP6G8JMa7o7lXvnz3GFKzPxzJdPfGK" crossorigin="anonymous"></script>
```
Comment (dipesh-rawat) [inline layouts/shortcodes/cncf-landscape.html:60]: The `integrity` hash value added in the script tag seems to be incorrect _(along with a typo)_, as I noticed that generating integrity hash locally yields a different value. It appears that the value was simply copied from an existing **_jquery-3.6.0.min.js_**. I suggest trying the following:

```suggestion
  <script src="https://cdnjs.cloudflare.com/ajax/libs/iframe-resizer/4.3.9/iframeResizer.min.js" integrity="sha384-hHTwgxzjpO1G1NI0wMHWQYUxnGtpWyDjVSZrFnDrlWa5OL+DFY57qnDWw/5WSJOl" crossorigin="anonymous"></script>
```

**Review by dipesh-rawat — COMMENTED** [review]
Thanks!

/approve

**Comment by dipesh-rawat** [comment]
/label tide/merge-method-squash


@dev-johnn If you're able to squash all the changes into 1 commit, that would be great (it makes things cleaner). But don't worry if you're not sure how to do it - learning Git can be challenging for sure.

**Comment by niranjandarshann** [comment]
@dev-johnn  Thank you for your contribution. It looks good to me. 
/lgtm

**Pull request merged**